gitlab/resource_gitlab_project.go

```
@@ -196,5 +197,16 @@ func resourceGitlabProjectDelete(d *schema.ResourceData, meta interface{}) error
 	log.Printf("[DEBUG] Delete gitlab project %s", d.Id())
 
 	_, err := client.Projects.DeleteProject(d.Id())
-	return err
+	if err != nil {
+		return err
+	}
+	for {
+		_, response, err := client.Projects.GetProject(d.Id())
+		if err != nil {
+			if response.StatusCode == 404 {
+				return nil
+			}
```

do we not want to return this error if it's not a 404?